src/components/permissions/routes.ts

```diff
@@ -135,15 +161,10 @@ const routeUserPermissions = async (request: any, reply: any) => {
     )
   )
 
-  const availablePermissions = Object.values(templatePermissionRows)
-    .filter(({ permissionName }) => !grantedPermissions.includes(permissionName))
-    .reduce((availablePermissions, { permissionName }) => {
-      if (!availablePermissions.includes(permissionName)) availablePermissions.push(permissionName)
-      return availablePermissions
-    }, [])
-
   return reply.send({
-    templatePermissions,
+    templatePermissions: templatePermissions.sort(({ name: aName }, { name: bName }) =>
+      aName.localeCompare(bName)
+    ),
```

Comment:
```suggestion
    templatePermissions,
```
It was really only the two text lists (granted/available) that needed sorting, since they're the ones that will actually appear in the UI (as checkboxes).